app/src/App.tsx

```diff
@@ -2,12 +2,18 @@ import React, { Suspense } from 'react';
 import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
 
 import ChoiceBoard from './routes/ChoiceBoard';
+import LikeBoard from './routes/LikeBoard';
 
 function App() {
   return (
     <main>
       <Router>
         <Switch>
+          <Route path="/likeMode">
+            <Suspense fallback="loading">
```

Comment:
I'm not very familiar with Suspense, sorry about that, but how do you expect it to work? Will we use Relay or Lazy imports? Just asking because of what I saw on these React docs:
- https://reactjs.org/docs/react-api.html#suspense
- https://reactjs.org/docs/concurrent-mode-suspense.html

Reply:
I think we can use a more traditional approach fetch-then-render approach now and then we will decide about using of not Relay (and Suspense) in the future. So, my suggestion is to remove `<Suspension/>`.